TrainingRNExpress/src/app/screens/BookDetails/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Text, View } from 'react-native';
+
+import { BOOKS } from '../../../constants';
+import { FIELDS } from './strings';
+
+import styles from './styles.js';
+
+class BookDetails extends Component {
+  state = { book: {}}
+
+  componentDidMount() {
+    const id = this.props.navigation.state.params.id;
+    this.setState({ book: BOOKS.find(book => book.id === id) })
+  }
+
+  static navigationOptions = ({ navigation }) => ({
+    title: BOOKS.find(book => book.id === navigation.state.params.id).title
```

Comment:
This component shouldn't access to the entire `BOOKS` information, since this one meant to be related by one in particular.
Maybe we can pass all the book info via nav param. What do you think ?